expressServer/scripts/generateUsernames.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+const { uniqueNamesGenerator, adjectives, colors, animals } = require('unique-names-generator');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+mongoose.connect(process.env.MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology: true })
+  .then(() => console.log('Connected to MongoDB'))
+  .catch(err => console.error('Could not connect to MongoDB', err));
+
+const userSchema = new mongoose.Schema({
+  username: String,
+  password: String,
+});
+
+const User = mongoose.model('User', userSchema);
+
+const generateRandomUsers = async (numUsers) => {
+  for (let i = 0; i < numUsers; i++) {
+    const username = uniqueNamesGenerator({ dictionaries: [adjectives, colors, animals] });
+    const password = uniqueNamesGenerator({ dictionaries: [colors] });
+
+    const user = new User({
+      username,
+      password,
+    });
+
+    await user.save();
```

Comment:
Here you do a save operation for each user, it means if you create 10000 users you will do 10000 database queries, knowing that the pricing for each query in a production environment can cost money, you are loosing money here. 

The best way to do this is to store arrays of n users and use the function insertMany: https://www.mongodb.com/docs/manual/reference/method/db.collection.insertMany/

Here you will need to balance, if the amount of users is too big, only one operation can cause performance problems in the database, you can test with different amount of users and decide how can you partition your array insertion